mlir/include/mlir/Dialect/LLVMIR/LLVMDialect.h

```diff
@@ -210,7 +210,7 @@ class GEPIndicesAdaptor {
 /// string (operations inserted at the builder insertion point).
 Value createGlobalString(Location loc, OpBuilder &builder, StringRef name,
                          StringRef value, Linkage linkage,
-                         bool useOpaquePointers);
+                         bool useOpaquePointers = true);
```

Comment:
I am guessing this will be removed incrementally.

Reply:
That's the plan, yes. We will be working on the LLVM dialect side once all users of typed pointers have been dropped.